packages/vite/src/node/plugins/css.ts

```diff
@@ -718,92 +718,90 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
         }
       }
 
-      if (!chunkCSS && !s) {
-        return null
-      }
```

Comment:
If a file contains `__VITE_CSS_URL__` (`!s === false`) and doesn't have side effect CSS imports (e.g. `import './foo.css'`) (`!chunkCSS === true`), the if-else block below should not run and `if (s)` block should run.